README.md

```diff
@@ -103,12 +103,17 @@ logging into Bluemix
 Now that you have your IBM ID and the `cf` command-line tool (see above),
 you can log into Bluemix and the deploy your app.
 
-First you should tell the `cf` command which environment you want to operate
+1. `bx login -sso -a http://api.ng.bluemix.net` (For people with @ibm.com accounts)
+2. Set target using `bx target --cf`
+3. Check api endpoint using `bluemix api`
+4. Push the project using `bx cf push`
+
+~~First you should tell the `cf` command which environment you want to operate
```

Comment:
What's with all the double tilde's - `~~` - in these lines.  The Markdown rendition just displays then as double tilde's, which is what I'd expect, but ... I don't think they're needed.

Reply:
The ~~ was for striking out the text. I thought I was making changes in my forked repo.

Reply:
No worries - you use a `~` prefix and suffix around the text you want to strike out.

Eg, to get this: "Here's some ~striked out~ text.", do this:
  
    Here's some ~striked out~ text.

But there's no sense striking out something that you want deleted; just delete it.  It'll show as deleted in the git diffs.

And yes, you are making changes in your forked repo, but I guess you created a PR in the primary repo pointing to your branch - which is the way you usually want to be doing it.  Perfect!